Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -16,10 +17,17 @@ namespace Microsoft.MixedReality.Toolkit.Utilities
     [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Rendering/ClippingPrimitive.html")]
     public abstract class ClippingPrimitive : MonoBehaviour, IMaterialInstanceOwner
     {
+        private const int InitialCollectionSize = 256;
```

Comment:
Might be good to sanity check how big a 256 member Dictionary is. Might be a bit wasteful for most use cases? But might be fine since these are all reference types.

Reply:
This value has been chosen with a reserve in mind (around 2-times). Depending on the button visuals complexity level it should be enough for around 10 buttons. In our application, lists are currently limited to 5 buttons. The most common size of the collection is between 80-100 items. So yes - it is a little bit wasteful, although while deciding the initial size of the collection I didn't want to only adhere to our solution.